src/Console/Commands/EmitEvents.php

```diff
@@ -32,15 +31,12 @@ class EmitEvents extends Command
 
     protected $signature = 'event-sourcing:emit
         {--dbConnection=mysql : Indicate the database connection to use (MySQL unbuffered for better performance when large amount of events)}
-        {--batchSize=100 : Indicate the amount of events to be sent per publish. Increase for higher throughput}
-        {--allEvents : Option to send all the events from the beginning by resetting the cursor}';
```

Comment:
Could you update the documentation to remove the allEvents option? Ensure the documentation does not say anything incorrect about the emit events feature.